examples/DisableShrinkingTest.php

```diff
@@ -7,6 +7,11 @@ class DisableShrinkingTest extends \PHPUnit_Framework_TestCase
     use TestTrait;
 
     /**
+     * @var int
```

Comment:
in tests, the coding convention I adopted for this repository is not to have field declarations, given that no one is at risk of accessing fiels from the test (which result to be `public` instead of `private`). Going to the root cause, how can I express conventions? Is there some `php-cs-fixer` template to start from?